cns/hnsclient/hnsclient_windows.go

```diff
@@ -426,6 +433,31 @@ func configureAclSettingHostNCApipaEndpoint(
 				return nil, err
 			}
 		}
+
+	}
+
+	if ncRequestedPolicies != nil {
+		// Iterate thru the requested endpoint policies where policy type is ACL, endpoint type is APIPA
+		// include the raw json message in the endpoint policies
+		for _, requestedPolicy := range ncRequestedPolicies {
+
+			if strings.EqualFold(requestedPolicy.Type, aclPolicyType) && strings.EqualFold(requestedPolicy.EndpointType, apipaEndpointType) {
```

Comment:
I believe you need to check if requestedPolicy is nil before accessing the members.